src/rocket_log.py

```diff
@@ -0,0 +1,27 @@
+import pandas as pd
+
+class RocketLog:
+  def __init__(self) -> None:
+
+    self.data = pd.DataFrame(columns=[
+      'time', # time
+      'rocket_component_id', # Unique rocket component identifier
+      'x', # x position
+      'y', # y position
+      'v_magnitude', # magnitude of velocity
+      'v_alpha', # angle of velocity
+      'v_direction', # direction of velocity
+      'v_x', # x value of velocity
```

Comment:
mathematically this is a fine name, for coding maybe rename to v_x_component

Reply:
Changed it to velocity_<component_name>